### PR TITLE
updated lightwood docs

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
## Description

updated lightwood docs

relates to https://github.com/mindsdb/mindsdb/issues/8477

Fixes #issue_number

## Type of change

- [ ] 📄 This change is a documentation update

### CODE REVIEW

File: docs/integrations/ai-engines/lightwood.mdx [hunk 1]
```diff
@@ -142,6 +142,54 @@ USING
...
+
+<AccordionGroup>
+
+<Accordion title="Regression">
+In the case of regression models, the `target_explain` column contains the following information:
+
+```
+{"predicted_value": 2951, "confidence": 0.99, "anomaly": null, "truth": null, "confidence_lower_bound": 2795, "confidence_upper_bound": 3107}
+```
+
```
Comment (paxcema) [inline docs/integrations/ai-engines/lightwood.mdx:163]: Let's add:

> The upper and lower bounds are determined via conformal prediction, and correspond to the reported confidence score (which can be modified by the user).

File: docs/integrations/ai-engines/lightwood.mdx [hunk 2]
```diff
@@ -142,6 +142,54 @@ USING
                 ]};
 ```
 
+## Predictions' Explanation
```
Comment (paxcema) [inline docs/integrations/ai-engines/lightwood.mdx:145]: Nit: maybe rename section to "Explainability"

File: docs/integrations/ai-engines/lightwood.mdx [hunk 3]
```diff
@@ -52,7 +52,7 @@ The values vary between 0 and 1, where 1 indicates a perfect predictor, based on
 You can check accuracy values for models using the [`DESCRIBE`](/sql/api/describe) statement.
 </Note>
 
-## Tuning the Lightwood ML Engine Features
+## Parameters' Tuning
```
Comment (paxcema) [inline docs/integrations/ai-engines/lightwood.mdx:55]: Nitpick: change to `Tuning the Lightwood ML Engine`? I think it sounds a bit more aligned with the content that follows.

File: docs/integrations/ai-engines/lightwood.mdx [hunk 4]
```diff
@@ -142,6 +142,54 @@ USING
...
+
+<Accordion title="Classification">
+In the case of classification models, the `target_explain` column contains the following information:
+
+```
+{"predicted_value": "No", "confidence": 0.6629213483146067, "anomaly": null, "truth": null, "probability_class_No": 0.8561, "probability_class_Yes": 0.1439}
+```
+
+<Note>
+The `confidence` score is produced by the conformal prediction module and is statistically valid. On the other hand, the `probability_class` comes directly from the model logits, which may be uncalibrated. Therefore, this `confidence` score is not right, on average - it may be optimistic or pessimistic, for instance.
```
Comment (paxcema) [inline docs/integrations/ai-engines/lightwood.mdx:175]: Corrections:

- Let's replace "and is statistically valid" by "and is well-calibrated".
- Let's replace the last phrase (starting with "Therefore, this `confidence` score...") by "Therefore, the `probability_class` score may be optimistic or pessimistic, i.e. coverage is not guaranteed to empirically match the reported score."

**Review by paxcema — CHANGES_REQUESTED** [review]

File: docs/integrations/ai-engines/lightwood.mdx (hunk 2 above)
Comment (martyna-mindsdb) [reply docs/integrations/ai-engines/lightwood.mdx:145]: "Predictions' Explainability" or just "Explainability"?

Comment (paxcema) [reply docs/integrations/ai-engines/lightwood.mdx]: Just "Explainability", but not feeling too strong either way

**Review by paxcema — APPROVED** [review]
